packages/medusa/src/utils/transaction/distributed-transaction.ts

```diff
@@ -0,0 +1,114 @@
+import { TransactionFlow, TransactionHandlerType, TransactionState } from "."
```

Comment:
suggestion(non-blocking, this-file): The public API of this util could benefit from having JSDoc, as it would otherwise be hard to use - both internally and externally.